packages/core/src/index.ts

```diff
@@ -1 +1,2 @@
 export { Civil } from "./civil";
+export { OwnedAddressTCRWithAppeals } from "./contracts/ownedAddressTCRWithAppeals";
```

Comment:
Civil should be the only and single entry point. the SC wrapper needs web3wrapper anyways, no use exporting it.